forge-gui/res/cardsfolder/upcoming/vault_11_voters_dilemma.txt

```diff
@@ -0,0 +1,13 @@
+Name:Vault 11: Voter's Dilemma
+ManaCost:2 W B
+Types:Enchantment Saga
+K:Chapter:3:DBToken,DBVote,DBVote
+SVar:DBToken:DB$ RepeatEach | RepeatPlayers$ Player.Opponent | RepeatSubAbility$ DBTokenBis | ChangeZoneTable$ True | SpellDescription$ For each opponent, you create a 1/1 white Human Soldier creature token.
+SVar:DBTokenBis:DB$ Token | TokenAmount$ 1 | TokenScript$ w_1_1_human_soldier
+SVar:DBVote:DB$ Vote | Defined$ Player | Secret$ True | StoreVoteNum$ True | VoteCard$ Creature | UpTo$ True | RememberVotedObjects$ True | SubAbility$ DBBranch | SpellDescription$ Each player secretly votes for up to one creature, then those votes are revealed. If no creature got votes, each player draws a card. Otherwise, destroy each creature with the most votes or tied for most votes.
```

Comment:
shouldn't you use VoteSubAbility?

Reply:
Hmm, I guess we need to check first if no player choose a creature ?

Reply:
Don't think I got it right yet

Reply:
should work already but extra Remembered isn't needed

Reply:
Hmm how should the check than work with the draw. is there a way to count the amount of votes cast ?

Reply:
the X should be correct already?

Reply:
there shouldn't be a reason for StoreVoteNum$ either